docs/merged_objects.md

```diff
@@ -46,6 +46,186 @@ rownames(merged_sce) # matrix row names
+# Access individual library metadata for SCPCL000000
+metadata(merged_sce)$library_metadata$SCPCL000000
+```
+
+Each such list will contain the following fields:
+
+
+| Item name           | Contents                                                                                                                       |
+| ------------------- | ------------------------------------------------------------------------------------------------------------------------------ |
+| `salmon_version`    | Version of `salmon` used for initial mapping                                                                                   |
+| `reference_index`   | Transcriptome reference file used for mapping                                                                                  |
+| `total_reads`       | Total number of reads processed by `salmon`                                                                                    |
+| `mapped_reads`      | Number of reads successfully mapped                                                                                            |
+| `mapping_tool`      | Pipeline used for mapping and quantification (`alevin-fry` for all current data in ScPCA)                                      |
+| `alevinfry_version` | Version of `alevin-fry` used for mapping and quantification                                                                    |
+| `af_permit_type`    | `alevin-fry generate-permit-list` method used for filtering cell barcodes                                                      |
+| `af_resolution`     | `alevin-fry quant` resolution mode used                                                                                        |
+| `usa_mode`          | Boolean indicating whether quantification was done using `alevin-fry` USA mode                                                 |
+| `af_num_cells`      | Number of cells reported by `alevin-fry`                                                                                       |
+| `tech_version`      | A string indicating the technology and version used for the single-cell library, such as 10Xv2, 10Xv3, or 10Xv3.1              |
+| `assay_ontology_term_id` | A string indicating the [Experimental Factor Ontology](https://www.ebi.ac.uk/ols/ontologies/efo) term id associated with the `tech_version`  |
+| `seq_unit`         | `cell` for single-cell samples or `nucleus` for single-nucleus samples                                                          |
+| `transcript_type`   | Transcripts included in gene counts: `spliced` for single-cell samples and `unspliced` for single-nuclei                       |
+| `sample_metadata`   | Data frame containing metadata for each sample included in the library (see the [`Sample metadata` section below](#singlecellexperiment-sample-metadata)) |
+| `miQC_model`        | The model object that `miQC` fit to the data and was used to calculate `prob_compromised`. Only present for `filtered` and `processed` objects |
```

Comment:
I would remove every reference to `Only present for x objects`. There's only one type of merged object.

Reply:
🤦‍♀️